pkg/providers.bzl

```diff
@@ -89,3 +89,19 @@ PackageFilegroupInfo = provider(
         "pkg_symlinks": "list of tuples of (PackageSymlinkInfo, origin)",
     },
 )
+
+PackageSubRPMInfo = provider(
```

Comment:
This seems like something only of interest between rpm rules, so it could be private to rpm_pfg.bzl.

Reply:
Sure.  Sounds good to me.